nautobot/docs/additional-features/jobs.md

```diff
@@ -29,7 +29,7 @@ Jobs may be installed in one of three ways:
 
 In any case, each module holds one or more Jobs (Python classes), each of which serves a specific purpose. The logic of each job can be split into a number of distinct methods, each of which performs a discrete portion of the overall job logic.
 
-For example, we can create a module named `devices.py` to hold all of our jobs which pertain to devices in Nautobot. Within that module, we might define several jobs. Each job is defined as a Python class inheriting from `extras.jobs.Job`, which provides the base functionality needed to accept user input and log activity.
+For example, we can create a module named `devices.py` to hold all of our jobs which pertain to devices in Nautobot. Within that module, we might define several jobs. Each job is defined as a Python class inheriting from `extras.jobs.Job`, which provides the base functionality needed to accept user input and log activity. Make sure you are _not_ inheriting `extras.jobs.models.Job` instead, otherwise Django will think you want to define a new database model.
```

Comment:
Thank you for this. I think this would be much better as a `warning` admonition:
```suggestion
For example, we can create a module named `devices.py` to hold all of our jobs which pertain to devices in Nautobot. Within that module, we might define several jobs. Each job is defined as a Python class inheriting from `extras.jobs.Job`, which provides the base functionality needed to accept user input and log activity.

!!! warning
    Make sure you are _not_ inheriting `extras.jobs.models.Job` instead, otherwise Django will think you want to define a new database model.
```